fixfasta.py

```
@@ -0,0 +1,446 @@
+
+from __future__ import annotations
+import sys
+import re
+import fileinput
+import argparse
+import logging
+from dataclasses import dataclass
+from textwrap import wrap
+from typing import List, Tuple, Optional, Iterator, Dict
+from collections import defaultdict
+from pathlib import Path
```

Ruff (0.11.9)

`pathlib.Path` imported but unused

Remove unused import: `pathlib.Path`

(F401)